### PR TITLE
LRQA-82667

### PR DESCRIPTION
**Pull request by Claudioss23**

### CODE REVIEW

**Comment by Claudioss23** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyBlogTests.testcase [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		}
+
+		task ("And Blog Entry 2 should exist") {
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blog Entry Content 2",
+				entryTitle = "Blog Entry Title 2");
+		}
+	}
+
+}
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyBlogTests.testcase:76]: Hi, remember our chat about variables? Can you create two variables one of the blog entry 1 and one for blog entry 2?

**Review by manuferreira — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyBlogTests.testcase (hunk 1 above)
Comment (Claudioss23) [reply portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyBlogTests.testcase:76]: Okay, I'll do it. Thanks for the review!

**Comment by Claudioss23** [comment]
ci:test:sf

**Comment by Claudioss23** [comment]
ci:test:sf

**Review by manuferreira — APPROVED** [review]
LGTM. Thanks for the update! :muscle:

**Pull request closed without merging**